modules/gapi/src/backends/oak/goakbackend.cpp

```diff
@@ -0,0 +1,616 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html.
+//
+// Copyright (C) 2021 Intel Corporation
+
+#include <opencv2/gapi/gkernel.hpp> // GKernelPackage
+
+#ifdef WITH_OAK_BACKEND
+
+#include <cstring>
+#include <unordered_set>
+
+#include <api/gbackend_priv.hpp>
+#include <backends/common/gbackend.hpp>
+
+#include "depthai/depthai.hpp"
+
+#include <opencv2/gapi/oak/oak.hpp>
+#include "oak_media_adapter.hpp"
+
+namespace cv { namespace gimpl {
+
+// Forward declaration
+class GOAKContext;
+
+class GOAKExecutable final: public GIslandExecutable {
+    friend class GOAKContext;
+    virtual void run(std::vector<InObj>&&,
+                     std::vector<OutObj>&&) override {
+        GAPI_Assert(false && "Not implemented");
+    }
+
```

Comment:
This overload is not mandatory as far as I know so you can just drop it. Or am I wrong here

Reply:
Unfortunately, it is

Reply:
Wow, ok